src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -308,20 +311,30 @@ class HMPPSDomainEventsEmitter(
       .withAdditionalInformation("enteredById", event.enteredById)
       .withAdditionalInformation("personId", event.personId)
 
-  private fun toVisitorRestriction(event: VisitorRestrictionOffenderEvent): HmppsDomainEvent =
+  private fun VisitorRestrictionOffenderEventUpserted.toDomainEvents(): List<HmppsDomainEvent> = listOf(
+    this.toDomainEvent(eventType = "prison-offender-events.visitor.restriction.upserted", description = "A prisoner visitor restriction record has been added or amended"),
+    this.toDomainEvent(eventType = "prison-offender-events.visitor.restriction.changed", description = "A prisoner visitor restriction record has changed"),
+  )
```

Comment:
is the changed event staying or a temporary bridge to the new events?

Reply:
only after VSIP switch to the new events - and they have requested them